doc/setting_up_vcpkg.md

```diff
@@ -2,6 +2,43 @@
 
 This document describes how to setup vcpkg to build applications using Microsoft Azure IoT device C SDK. It demonstrates building and running a C language SDK sample for Windows, Linux, and Mac.
 
+## C SDK vcpkg support
+
+The C SDK uses [vcpkg](https://github.com/microsoft/vcpkg) primarily for LTS releases. The most recent LTS release can be installed using the syntax: `vcpkg install azure-iot-sdk-c` and builds using the following CMake flags:
+
+```
+    -Dskip_samples=ON
+    -Duse_installed_dependencies=ON
+    -Duse_default_uuid=ON
+    -Dbuild_as_dynamic=OFF
+    -Duse_edge_modules=ON
+    -Dwarnings_as_errors=OFF
+```
+
+The `use-prov-client` feature uses the syntax: `vcpkg install azure-iot-sdk-c[use-prov-client]` and builds using additional CMake flags:
```

Comment:
Sorry for tag if you spoke offline, but @cipop could you please double check that these flags still make sense in light of #2229?  If not very well need a vcpkg PR, but since it's on mind wanted to ask.  Thanks